numpy/f2py/tests/test_array_from_pyobj.py

```diff
@@ -46,7 +46,7 @@ def flags_info(arr):
 
 def flags2names(flags):
     info = []
-    for flagname in ['CONTIGUOUS', 'FORTRAN', 'OWNDATA', 'ENSURECOPY',
+    for flagname in ['CONTIGUOUS', 'F_CONTIGUOUS', 'OWNDATA', 'ENSURECOPY',
```

Comment:
We should probably use `C_CONTIGUOUS` instead of `CONTIGUOUS`, or at least test both.